test/integration/storageversionmigrator/util.go

```diff
@@ -768,15 +745,30 @@ func (svm *svmTest) updateCRD(
 		t.Fatalf("Failed to update CRD: %v", err)
 	}
 
-	crd, err := svm.apiextensionsclient.ApiextensionsV1().CustomResourceDefinitions().Get(ctx, crdName, metav1.GetOptions{})
+	// wait for CRD to be updated with desired versions
```

Comment:
This is checking the wrong thing.  What you need to check is that the Kube API server has caught up to the config change _in memory_.